editor/loadFile.js

```diff
@@ -0,0 +1,26 @@
+function createFileLoader (moonchild, channel) {
+  var fileLoader = Object.create(null);
+
+  // assuming that the editor doesn't get changed
+  // weird stuff will happen if the editor -does- get changed
+  var editor = moonchild.getEditor()._codeMirror;
+  var filePath;
+
+  channel.on("fileLoad", function (messageData) {
+    // assuming JSON structure of form
+    // {
+    //   ...
+    //   content: "",
+    //   filePath: "bla.js"
+    //   ...
+    // }
+
+    var text = messageData.content;
+    filePath = messageData.filePath;
+
+    editor.setValue(text);
+
+    // let the editor know that a change happened
+    moonchild.poke();
```

Comment:
I don't mind leaving it separate, but given how little code there is (and it's unlikely to get bigger), I would personally just move it in to the editor itself. But it's up to you.